### ISSUE TITLE
Support multiple simultaneous okta role executions

### ISSUE THREAD

**Issue by eldondevat** [segmentio/aws-okta#98]
This is a bit out there perhaps, and forgive me if this is somehow already supported and I have simply missed it, but in some scenarios, people might be setting up activities involving resource in multiple accounts, and require the availability of multiple simultaneous aws-okta profiles when performing an `exec`.
 In the case I am considering, there is no aws cross-account role assumption, but rather, each aws account has its own SAML endpoint, and so it would be necessary to utilize several different sets of environment variables for this. I could imagine something along the lines of the following:
`aws-okta exec prof-a,prof-b -- terraform plan` .
 Then, the execution environment might contain, for example, `AWS_ACCESS_KEY_ID_prof_a` and `AWS_ACCESS_KEY_ID_prof_b` without necessarily having to enable AWS role assumption from a to b or b to a, and the underlying tool (in my case terraform) could be directed to use the appropriate environment variables.

**Comment by eldondevat** [comment]
Yeah, I had considered shell. I didn't know how popular it would be, preferred implementing in go, but I will try this out. Thanks for the feedback!

**Comment by nickatsegment** [comment]
I mean, you can do it in whatever language you'd like :) I'm just saying getting it merged into aws-okta isn't likely to happen. Every feature we add has a maintenance burden, and I'm not sure this one is worth it. Hope that makes sense

**Comment by eldondevat** [comment]
Yep, understand for sure, happy to fork and possibly contribute a fix, but I won't bother to polish to a PR sheen if it's not of interest. Thanks for the excellent tool, btw!

**Comment by nickatsegment** [comment]
These sorts of little helper scripts pop up all the time (also with our other project `chamber`). I wonder if there's a need for some "helper scripts" repo or something

**Comment by gitteri** [comment]
@eldondevat did you come up with a solution to this? I have the same problem and have hacked together a solution but it's not quite there. I'm running into issues with expired security tokens when trying to run terraform. @nickatsegment would you consider adding a force refresh command to rotate keys on demand?

**Comment by gitteri** [comment]
actually, I figured it out!
